personal_notion_bot/telegram_side/main.py

```diff
@@ -0,0 +1,12 @@
+from loader import bot, storage
+
+async def on_shutdown(dp):
+	await bot.close()
+	await storage.close()
+
+if __name__ == '__main__':
+	from aiogram import executor
+	import handlers
+
+	executor.start_polling(handlers.dp, on_shutdown = on_shutdown, skip_updates=True)
```

Comment:
There are lots of pep-8 style inconsistencies. 

The "hypermodern" python best practice is to use "Black linter": https://black.readthedocs.io/en/stable/. 

If you configure it properly, it will fix many style issues on saving a file. 

Here is how to configure it in PyCharm: https://black.readthedocs.io/en/stable/integrations/editors.html